website/docs/enterprise/pipelines/getting-started.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Getting started
+sidebar_position: 1
+hide_title: true
+---
+
+import TierLabel from "../../_components/TierLabel";
+
+import CodeBlock from "@theme/CodeBlock";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+# Getting started with pipelines <TierLabel tiers="enterprise" />
+
+## Requirements
+- You have configured Weave Gitops Enterprise [RBAC for pipelines](../authorization)
+
+## Define a pipeline
+
+A pipeline allows you to define the route your application is taking in order to make it to production.
+There are three main concepts playing in a pipeline:
+- the `application` to deliver
+- the `environments` that your app will go through in its way to production (general)
+- the `deployment targets` or the clusters that each environment has
+
+In weave gitops you could define a delivery pipeline using a `Pipeline` Custom Resource Definitions.
+An example of how it looks for an application `podinfo` is shown below.
+
+```yaml
+apiVersion: pipelines.weave.works/v1alpha1
+kind: Pipeline
+metadata:
+  name: podinfo-02
+  namespace: default
+spec:
+  appRef:
+    apiVersion: helm.toolkit.fluxcd.io/v2beta1
+    kind: HelmRelease
+    name: podinfo
+  environments:
+    - name: dev
+      targets:
+        - namespace: podinfo-02-dev
+          clusterRef:
+            kind: GitopsCluster
+            name: dev
+            namespace: flux-system
+    - name: test
+      targets:
+        - namespace: podinfo-02-qa
+          clusterRef:
+            kind: GitopsCluster
+            name: dev
+            namespace: flux-system
+        - namespace: podinfo-02-perf
+          clusterRef:
+            kind: GitopsCluster
+            name: dev
+            namespace: flux-system
+    - name: prod
+      targets:
+        - namespace: podinfo-02-prod
+          clusterRef:
+            kind: GitopsCluster
+            name: prod
+            namespace: flux-system
+```
+
+In the previous example, `podinfo` application is delivered to a traditional pipeline composed by `dev`, `qa`, (`perf`) and `production` environments.
```

Comment:
Not sure why `perf` needs to be wrapped in round brackets.

Reply:
remove the wrapping